pkg/template/template.go

```diff
@@ -0,0 +1,84 @@
+package template
+
+import (
+	"bytes"
+	"embed"
+	"io"
+	"io/fs"
+	"text/template"
+
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+// TemplateVariables contains all the available variables that are supported by the templates
+type TemplateVariables struct {
```

Comment:
the package name is `template` so you could rename the type by cutting the prefix to:
```suggestion
type Variables struct {
```

Reply:
Good point, done in https://github.com/codeready-toolchain/toolchain-common/pull/365/commits/f08be69fbfcb8403531f12252c2480366dcffb4e